com.unity.multiplayer.mlapi/Runtime/Timing/NetworkTime.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace MLAPI.Timing
+{
+    /// <summary>
+    /// A struct to represent a point of time in a networked game.
+    /// Time is stored as a combination of amount of passed ticks + a duration offset.
+    /// This struct is meant to replace the Unity <see cref="Time"/> API for multiplayer gameplay.
+    /// </summary>
+    public struct NetworkTime
+    {
+        private double m_Time;
```

Comment:
this might be a controversial take but I generally expect put units as suffixes to variables which are not obvious by their type. for instance, `m_Time` is a `double` but it's not clear whether it's seconds, minutes or milliseconds. My gut tells me it's seconds but it's not immediately obvious. I know Unity also did `float Time.time` and all that without `Sec` suffix but maybe we could to `m_TimeSec` internally and `TimeSec` on public APIs.
minor, nit-picking but I'm curious what you'd think.

Reply:
I totally agree about adding a unit suffix at places where it's needed for clarity. So yeah I'll change this to `m_TimeSec`.

I don't think that changing the public API is a good idea here. I think Unity users are already used to Time.time being in seconds and we should just follow that convention because else it could become confusing as to why we specify the units but the regular time API does not.